tests/manage/z_cluster/cluster_expansion/test_add_node_crashcollector.py

```diff
@@ -0,0 +1,136 @@
+import logging
+
+from ocs_ci.framework.testlib import tier2, ignore_leftovers, ManageTest, bugzilla
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
+from ocs_ci.ocs.resources.pod import get_pod_obj, get_all_pods
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import OPENSHIFT_STORAGE_NAMESPACE
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_pod_node, get_mgr_pods
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@ignore_leftovers
+@bugzilla("1898808")
+@skipif_openshift_dedicated
+class TestAddNodeCrashCollector(ManageTest):
+    """
+    Add node with OCS label and verify crashcollector created on new node
+
+    Test Procedure:
+    1.Get worker node where mgr pod running [worker-node-x]
+    2.Add worker node with OCS label
+    3.Check ceph status [wait_for_rebalance]
+    4.Drain worker-node-x
+    5.Wait for 3 mon pods to be on running state
+    6.Verify ceph-crashcollector pod running on worker node where "rook-ceph" pods are running.
+    7.Schedule worker-node-x
+    8.Wait for 3 osd pods to be on running state
+    9.Verify ceph-crashcollector pod running on worker node where "rook-ceph" pods are running.
+
+    """
+
+    def test_add_node_crash_collector(self, add_nodes, node_drain_teardown):
+        """
+        Add node with OCS label and verify crashcollector created on new node
+
+        """
+        logger.info("Get Node name where mgr pod running")
+        mgr_pod_nodes = [get_pod_node(pod) for pod in get_mgr_pods()]
+        mgr_pod_node_name = [node.name for node in mgr_pod_nodes]
+
+        logger.info("Add one worker node with OCS label")
+        add_nodes(ocs_nodes=False, node_count=1)
+        ceph_cluster_obj = CephCluster()
+        assert ceph_cluster_obj.wait_for_rebalance(
+            timeout=3600
+        ), "Data re-balance failed to complete"
+
```

Comment:
Lets also add a check verifying that crashcollector pod is not running on the newly added node